Service-Broker-Setup.md

```diff
@@ -13,50 +13,66 @@ a CAP cluster.
 
 - Helm must be configured; see [helm docs] if you need assistance.
 - A working CAP deployment.
+- External databases must be reachable from the running applications; please
+  refer to [application security groups] for details.
 
 [helm docs]: https://docs.helm.sh/using_helm/#quickstart
+[application security groups]: http://docs.cloudfoundry.org/concepts/asg.html
 
 # Deploying the MySQL chart
 
-1. You need an external MySQL installation, with account credentials that allow
-    creating and deleting both databases and users.
-
-1. Configure the database access credentials.
-    ```bash
-    export DBHOST=…
-    export DBPORT=…
-    export DBUSER=…
-    export DBPASS=…
-    ```
-    The necessary credentials are:
+You need an external MySQL installation, with account credentials that allow creating and deleting both databases and users.
+
+## Configuring the deployment
+
+Create a values.yaml file (the rest of the document assumes it is called `usb-config-values.yaml`) with the settings required for the install.  Use the file below as a template, and modify the values to suit your installation.
+
+```yaml
+env:
+  # Database access credentials; the given user must have privileges to create
+  # delete both databases and users
```

Comment:
`create __and__ delete`. The `and` is missing in the sentence.

Reply:
Fixed in 9bdb11490e74fdbd3a616b863807e1bb7934077b